opentelemetry-api/src/opentelemetry/trace/propagation/textmap.py

```diff
@@ -20,7 +20,64 @@
 TextMapPropagatorT = typing.TypeVar("TextMapPropagatorT")
 
 Setter = typing.Callable[[TextMapPropagatorT, str, str], None]
-Getter = typing.Callable[[TextMapPropagatorT, str], typing.List[str]]
+
+
+def default_get(carrier: TextMapPropagatorT, key: str) -> typing.List[str]:
+    return [carrier.get(key)] if carrier.get(key) else []
+
+
+def default_keys(carrier: TextMapPropagatorT) -> typing.List[str]:
+    return list(carrier.keys())
+
+
+GetterGetFunction = typing.Callable[
+    [TextMapPropagatorT, str], typing.List[str]
+]
+GetterKeysFunction = typing.Callable[[TextMapPropagatorT], typing.List[str]]
```

Comment:
Having `Function` as a suffix is misleading as these are just types. Maybe just `GetterKeys` and `GetterGet`.